### PR TITLE
Clean ehr export dataset r

### PR DESCRIPTION
**Pull request by myyong**

### CODE REVIEW

File: client/myexploratory.Rmd [hunk 1]
```diff
@@ -0,0 +1,4 @@
+```{python}
+
+```
+
```
Comment (tpetricek) [inline client/myexploratory.Rmd]: I'm not sure what this file does - can we remove it?

File: client/src/services/documentService.ts [hunk 2]
```diff
@@ -7,7 +7,8 @@ interface DocumentElement {
 
 async function getSampleDocument(): Promise<Array<DocumentElement>> {
     async function getDocumentMd() : Promise<string> {
-      let response = await axios.get("/sample.md")
+      let response = await axios.get("/cleanEHR.md")
```
Comment (tpetricek) [inline client/src/services/documentService.ts]: We should figure out how to turn this into a parameter of some kind. Could we perhaps pass the file name in the query string, i.e. you'd navigate to `http://localhost:8080/?cleanEHR` and we'd parse the query in the URL and load the right file?

File: client/myexploratory.Rmd (hunk 1 above)
Comment (myyong) [reply client/myexploratory.Rmd]: Done in https://github.com/wrattler/wrattler/commit/688bbfb33cee10f15647a250a52b5a2dafb8c7ca

File: client/src/services/documentService.ts (hunk 2 above)
Comment (myyong) [reply client/src/services/documentService.ts]: Done in https://github.com/wrattler/wrattler/commit/688bbfb33cee10f15647a250a52b5a2dafb8c7ca

The way to specify the source code file is to use `http://localhost:8080/?sample` or `http://localhost:8080/?cleanEHR`

**Comment by tpetricek** [comment]
Sounds like this should go into master now, unless you have some reason for wanting to finish this in a branch?

**Comment by tpetricek** [comment]
Okay, this now has all the unmerged work from me and @myyong. 

What are the current issues that remain to be solved before we can merge this? Do we still need to figure out how to convert data frames to/from pandas when running Python? (Alternatively, can we insert `pd.DataFrame` into appropriate places in the code to make it work for now - and we can remove that later?)

**Comment by nbarlowATI** [comment]
from my point of view i think this can be merged - it includes some effort in the python service to convert things to/from dataframes, and it improves the handling of NaN/None/NULL/NA/null across the two services..  
there is another branch which improves error handling and a first go at text output in the python service, but that can be another PR i think..

**Pull request merged**